vault/provider.go

```diff
@@ -173,6 +173,14 @@ var (
 			Resource:      UpdateSchemaResource(pkiSecretBackendKeysDataSource()),
 			PathInventory: []string{"/pki/keys"},
 		},
+		"vault_transform_encode": {
```

Comment:
Looks like the tests are expecting an `encode_role` resource. We can either update this or update the tests, I think either should be fine :) 
```suggestion
		"vault_transform_encode_role": {
```

Reply:
Thanks! There was inconsistency here across the tests/docs and actual implementation. But the resource is [registered](https://github.com/hashicorp/terraform-provider-vault/blob/bff5e91e7289fc847e4fb4c2e2044b78ffb2aa76/generated/terraform_registry.go#L20) as `vault_transform_decode` so I will stick with that.